### DISCUSSION TITLE
App installation failed with error message

### DISCUSSION THREAD

**Discussion by jenkingaule** [microsoft/WindowsAppSDK#1726] [Q&A]
App installation failed with error message: Windows cannot install package cc5a0fee-f182-4d02-a177-3c2f002b4f34_0.9.6.0_x64__vn8kspbc310ge because this package depends on a framework that could not be found. Provide the framework "Microsoft.ProjectReunion.0.8" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 8005.278.2204.0, along with this package to install. The frameworks with name "Microsoft.ProjectReunion.0.8" currently installed are: {Microsoft.ProjectReunion.0.8_8004.256.1745.0_x64__8wekyb3d8bbwe Microsoft.ProjectReunion.0.8_8004.256.1745.0_x86__8wekyb3d8bbwe} (0x80073cf3)

**Comment by andrewleader** [comment]
Hi @LUCKYGorso, can you provide more info about what you're doing? Are you trying to install your app on another machine? How are you trying to install your app, through a local MSIX file or through the Store? What version of the NuGet package are you using?

**Comment by Polanich** [comment]
I had the same problem and had to roll back the three ProjectReunion packages to 0.8.4 to install successfully.

**Comment by giraudremy** [comment]
I have simular issue on Microsoft Store, I can't use 0.8.5 version :
[https://github.com/microsoft/WindowsAppSDK/discussions/1695](url)

**Comment by sowenzhang** [comment]
same issue when trying to upload to Store

![image](https://user-images.githubusercontent.com/2231315/140671599-1611d087-8995-4f3a-a512-0eda6e845550.png)

What should I do?

**Reply by andrewleader** [reply]
Hi @sowenzhang, we're investigating this issue. Subscribe to issue https://github.com/microsoft/WindowsAppSDK/issues/1749 to get updates on fixing being able to publish apps using 0.8.5 to the Store.
